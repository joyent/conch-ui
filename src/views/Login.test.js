/* global describe, it, expect */
import m from "mithril";
import stream from "mithril/stream";
import { tidy } from "mithril-jest";

import LoginView from "views/Login";

describe("Login component", () => {
	it("default title should contain 'Login'", () => {
        const Login = new LoginView();
		const cmp = m(Login);
		const html = tidy(cmp);
		expect(html).toContain("Login");
		expect(html).toMatchSnapshot();
	});
});
