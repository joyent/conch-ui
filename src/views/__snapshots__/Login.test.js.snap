// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component default title should contain 'Login' 1`] = `
"<section class=\\"hero is-fullheight\\">
  <div class=\\"hero-body\\">
    <div class=\\"container has-text-centered\\">
      <div class=\\"column is-4 is-offset-4\\">
        <div class=\\"box\\">
          <h3 class=\\"title\\">
            Login to Conch
          </h3>
          <form>
            <div class=\\"field\\">
              <div class=\\"control\\">
                <input type=\\"email\\" placeholder=\\"Email address\\"
                class=\\"input is-info is-fullwidth is-rounded\\">
              </div>
            </div>
            <div class=\\"field\\">
              <div class=\\"control\\">
                <input type=\\"password\\" placeholder=\\"Password\\"
                class=\\"input is-info is-fullwidth is-rounded\\">
              </div>
            </div><button type=\\"submit\\" class=
            \\"button is-primary is-fullwidth\\">Login</button>
          </form>
        </div>
      </div>
    </div>
  </div>
</section>
"
`;
